.github/workflows/check_vdb_command

```diff
@@ -9,6 +9,7 @@ parser.add_argument("--database_name", help="name of the postgres database", def
 parser.add_argument("--database_port", help="port of the postgres database", default="5432")
 parser.add_argument("--database_user", help="username of the postgres database user", default="postgres")
 parser.add_argument("--database_password", help="password of the postgres database user", default="postgres")
+parser.add_argument("--storagediffs_source", help="geth version", default="new-geth")
```

Comment:
I"m still kinda confused how this `new-geth` value works.  There's no chance we're inadvertently running the csv version of extract-diffs is there?

Reply:
You're right.. this was causing it to default to csv and the container would report that it was healthy and waiting for a csv file to appear. Changed back to `geth` and added a `wss://` path to the node.